apps/ai/src/app/api/receive/route.ts

```diff
@@ -55,7 +55,7 @@ export const POST = async (request: NextRequest) => {
         },
       },
     },
-    where: { email: envelope.from, credentials: { some: { appId: env.APP_ID } } },
+    where: { email: envelope.from, linkedBy: null, credentials: { some: { appId: env.APP_ID } } },
```

Comment:
Now relying on `findFirst` and not only email as a where condition but also `linkedBy´ to be `null` which means it's a parent user.